src/AudacityApp.cpp

```diff
@@ -400,6 +406,74 @@ void PopulatePreferences()
    gPrefs->Flush();
 }
 
+wxDialog* MakePluginsCheckResultsDialog(const std::vector<wxString>& failed)
+{
+   auto dialog = safenew wxDialogWrapper(nullptr, wxID_ANY, XO("New Plugins"));
```

Comment:
Does this leak?  A wxWindow is managed when it has a non-null parent, but this non-modal dialog does not!
But I see it does not leak because all the button event handlers cause it to destroy itself.  Very well, mention that here, to justify `safenew` (which is something like an assertion -- something needing a proof when you see it.)